tests/ap_tests/test_apnos.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+from configuration_data import TEST_CASES
+
+
+@pytest.mark.run(order=3)
```

Comment:
Why do we need a specific order for our test cases?

Reply:
Do we have a JIRA for order removal?